master/master.cfg

```diff
@@ -1219,7 +1222,7 @@ def distsnap_buildfactory(platform, channel_label):
     elif 'cross-linux' in platform or 'ios' in platform:
         command.append('dist')
     else:
-        command.append('distcheck')
+        command.append('dist') # FIXME distcheck
```

Comment:
We're talking on IRC about this, but probably want to revert this? Or maybe run `check` on other platforms?